applications/ehr_query_llm/fhir/README.md

```diff
@@ -134,14 +109,16 @@ From the menu, pick one of the choices for the resources of interest.
 
 ## Packaging the Application for Distribution and Deployment
 
-With Holoscan CLI, an applications built with Holoscan SDK can be packaged into a Holoscan Application Package (HAP), which is an [Open Container Initiative](https://opencontainers.org/) compliant image. An HAP is well suited to be distributed for deployment on hosting platforms, be it Docker Compose, Kubernetes, or else. Please refer to [Packaging Holoscan Applications](https://docs.nvidia.com/holoscan/sdk-user-guide/holoscan_packager.html) in the User Guide for more information.
+With Holoscan CLI, applications built with Holoscan SDK can be packaged into a Holoscan Application Package (HAP), which is an [Open Container Initiative](https://opencontainers.org/) compliant image. An HAP is well suited to be distributed for deployment on hosting platforms, be it Docker Compose, Kubernetes, or otherwise. Please refer to [Packaging Holoscan Applications](https://docs.nvidia.com/holoscan/sdk-user-guide/holoscan_packager.html) in the User Guide for more information.
 
 This example application provides all the necessary contents for HAP packaging. It is required to perform the packaging in a Python virtual environment, with the application's dependencies installed, before running the following script to reveal specific packaging commands.
+
 ```bash
 applications/ehr_query_llm/fhir/packageHAP.sh
```

Comment:
This shell script fails, due to the breaking change in the `--platform` option of `holoscan-cli`, so the needed change is
  - `PLATFORM=x64-workstation` -> `PLATFORM=x86_64` on line 32

Also, installing holoscan-cli (v3.3) in the venv met with conflict and it re-installed a lower version of pydantic of 1.10.22, causing the test client to fail. Workaround is having the test client run in the venv with only packages from `holohub/applications/ehr_query_llm/fhir/requirements.txt` while we can separately figure out the changes to be done to holoscan-cli to avoid the conflict.

Reply:
In the same venv, user would have to go back and forth with installing holoscan-cli and installing fhir-resources if repeating the steps. If running the CLI packaging once only, this workaround hides the issue.

Reply:
That's right! How frequent do you expect the user to switch between developing and packaging? Isn't the packaging usually the final step?

Anyways the issue will go away once `holoscan-cli` updates its dependencies in the future.